### PR TITLE
Improve LP pool setup, fix pricing section, add base sepolia

### PR DESCRIPTION
**Pull request by hieronx**
### Description

This pull request...

#<issue-number>

### Approvals

- [ ] Dev

### CODE REVIEW

**Comment by hieronx** [comment]
Note: this is how the buttons now look after pool setup:

Screenshot 2024-08-15 at 11 27 45

I couldn't figure out easily how to make the first button full width. Maybe @kattylucy you have a quick idea :)

This is https://github.com/centrifuge/apps/pull/2376/files#diff-d89a4e0925a9c0497f6da54ffd3a12567b7836d58f2c4cea67098c327d0597b0R146

File: centrifuge-app/src/pages/IssuerPool/Configuration/index.tsx [hunk 1]
```diff
@@ -39,6 +40,7 @@ function IssuerPoolConfiguration() {
           <Issuer />
           <EpochAndTranches />
           <LoanTemplates />
+          {isPoolAdmin && <OnboardingSettings />}
```
Comment (hieronx) [inline centrifuge-app/src/pages/IssuerPool/Configuration/index.tsx:43]: In a future ticket we will expand the Investors tab, this is just a first step at moving the onboarding settings (which are almost never updated) to the configuration tab.

File: centrifuge-app/src/pages/Loan/PricingValues.tsx [hunk 2]
```diff
@@ -33,12 +34,14 @@ export function PricingValues({ loan, pool }: Props) {
...
-
     const borrowerAssetTransactions = assetTransactions?.filter(
       (assetTransaction) => assetTransaction.asset.id === `${loan.poolId}-${loan.id}`
     )
-    const latestPrice = getLatestPrice(latestOraclePrice.value, borrowerAssetTransactions, pool.currency.decimals)
+    const latestPrice = getLatestPrice(latestOraclePrice, borrowerAssetTransactions, pool.currency.decimals)
+
+    const days = latestPrice.timestamp > 0 ? getAge(new Date(latestPrice.timestamp).toISOString()) : undefined
+
+    const accruedPrice = 'currentPrice' in loan && loan.currentPrice
```
Comment (hieronx) [inline centrifuge-app/src/pages/Loan/PricingValues.tsx:44]: TLDR of the changes: if linear pricing is enabled, we should show that (so it matches the latest price in the chart), and we should show the latest manual price update date. If there is a latest settlement price, we should show the date of date.

**Review by kattylucy — APPROVED** [review]
LGTM!

**Pull request merged**